lib/generators/boring/webmock/install/install_generator.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module Boring
+  module Webmock
+    class InstallGenerator < Rails::Generators::Base
+      desc "Adds webmock gem to the application"
+
+      SUPPORTED_TEST_FRAMEWORKS = %w[rspec minitest]
+
+      class_option :test_gem, type: :string,
+                   desc: "Tell us the framework you use for writing tests in your application. Supported options are #{SUPPORTED_TEST_FRAMEWORKS}"
+      
+      def verify_presence_of_supported_test_gem
+        test_gem = options[:test_gem]
+
+        raise(NotImplementedError, "test_gem option can't be blank! Supported test_gem options are #{SUPPORTED_TEST_FRAMEWORKS}") if test_gem.blank?
+
+        if SUPPORTED_TEST_FRAMEWORKS.exclude?(test_gem)
+          raise(NotImplementedError, "#{test_gem} is not supported as a test_gem option! Supported options are #{SUPPORTED_TEST_FRAMEWORKS}")
+        end
+
+        if test_gem.eql?('rspec')
+          verify_presence_of_test_gem('rspec-rails')
+        else
+          verify_presence_of_test_gem('minitest')
+        end
+      end
+
+      def add_webmock_gem
+        say "Adding webmock gem", :green
+        
+        gem_content = <<~RUBY
+          \n
+          \tgem "webmock"
+        RUBY
+
+        insert_into_file "Gemfile", gem_content, after: /group :test do/
+
+        Bundler.with_unbundled_env do
+          run "bundle install"
+        end
+      end
+
+      def configure_webmock
+        test_gem = options[:test_gem]
+
+        say "Configuring webmock", :green
+
+        if test_gem.eql?('rspec')
+          inject_into_file "spec/spec_helper.rb",
+                           "require 'webmock/rspec'\n\n",
+                           before: /\A/
+        elsif test_gem.eql?('minitest')
+          inject_into_file "test/test_helper.rb",
```

Comment:
I think we should include `disable_net_connect` by default. IMO, it adds value when using webmock.

```
inject_into_file 'test/test_helper.rb', before: 'class ActiveSupport::TestCase' do
      <<~RUBY
        require 'webmock/minitest'
        WebMock.disable_net_connect!(allow_localhost: true)
      RUBY
    end
```

Reply:
I agree, added now.